File: mcp/jcodemunch-mcp/tests/fixtures/php/sample.php
<?php

const MAX_RETRIES = 3;

/**
 * Authenticate a user token.
 */
function authenticate(string $token): bool
{
    return strlen($token) > 0;
}

/**
 * Manages user operations.
 */
class UserService
{
    /**
     * Get a user by ID.
     */
    public function getUser(int $userId): array
    {
        return ['id' => $userId];
    }

    /**
     * Create a new user.
     */
    public static function create(string $name): self
    {
        return new self();
    }
}

interface Authenticatable
{
    public function authenticate(string $token): bool;
}

trait Timestampable
{
    public function getCreatedAt(): string
    {
        return date('Y-m-d');
    }
}

enum Status
{
    case Active;
    case Inactive;
    case Pending;
}

File: mcp/jcodemunch-mcp/tests/fixtures/python/sample.py
MAX_RETRIES = 3

class UserService:
    """Manages user operations."""
    def get_user(self, user_id: int) -> dict:
        """Get user by ID."""
        return {"id": user_id}

    def delete_user(self, user_id: int) -> bool:
        """Delete a user."""
        return True

def authenticate(token: str) -> bool:
    """Authenticate a token."""
    return len(token) > 0
